nest-backend/src/controllers/post-like.controller.ts

```diff
@@ -0,0 +1,53 @@
+import { Controller, Delete, Get, NotFoundException, Param, Post } from '@nestjs/common'
+import { PostLikeService } from '../services/post-like.service'
+import { PostLike } from '../entities/post_like.entity'
+import { AuthorizedUser } from '../decorators/jwt.decorator'
+import { Account } from '../entities/account.entity'
+import { AllowAny } from '../decorators/allow-any.decorator'
+import { ApiTags } from '@nestjs/swagger'
+
+@ApiTags('Post-like')
+@Controller('post/like')
+export class PostLikeController {
+
+  constructor(
+    private readonly postLikeService: PostLikeService,
+  ) {
+  }
```

Comment:
Empty constructor. If you want to use it, please make it one line

```suggestion
  constructor(private readonly postLikeService: PostLikeService) { }
```